File: 242-valid-anagram/valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int freq[26]={0};
        for(char i : s)
            freq[i-'a']++;

        for(char i : t)
            freq[i-'a']--;
        
        for(int i: freq)
            if(i!=0) return 0;
        
        return 1;
        

    }
};

File: 242-valid-anagram/valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        freq=[0]*26
        for i in s:
            freq[ord(i)-ord('a')]+=1
        
        for i in t:
            freq[ord(i)-ord('a')]-=1

        for i in freq:
            if i!=0:
                return False
        return True
            
